tests/terraform/graph/checks/resources/GCPComputeFirewallOverlyPermissiveToAllTraffic/main.tf

```diff
@@ -0,0 +1,89 @@
+resource "google_compute_network" "example" {
+  name = "example"
+  auto_create_subnetworks = false
+}
+
+#case1 - PASS
```

Comment:
please also add 1 pass and 1 fail example for `google_compute_firewall_policy_rule`

Reply:
I have removed this resource - google_compute_firewall_policy_rule as - as the policy doesn't apply to it